app/src/main/java/org/wikipedia/search/RecentSearchesFragment.kt

```diff
@@ -155,4 +181,34 @@ class RecentSearchesFragment : Fragment() {
             holder.bindItem(pos)
         }
     }
+
+    private inner class NamespaceItemViewHolder constructor(itemView: View) : RecyclerView.ViewHolder(itemView), View.OnClickListener {
+        private var ns: Namespace? = null
+
+        fun bindItem(ns: Namespace?) {
+            itemView.setOnClickListener(this)
+            this.ns = ns
+            (itemView as TextView).text = if (ns == null) getString(R.string.search_namespaces) else namespaceMap[callback?.getLangCode()]?.get(ns).orEmpty() + ":"
```

Comment:
Would it be possible to set `ns == null` as a variable so that we don't have to add the logic in the following code?

Reply:
yep